### PR TITLE
Add Alpha phase

### PR DESCRIPTION
**Pull request by colugo**
I've transposed the content from the google-doc alpha guide

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <ul class="top-level">
   <li>
     {% if page.section == 'discovery' %}
-      {% include discovery.html %}}
+      {% include discovery.html %}
+    {% elsif page.section == 'alpha' %}
+        {% include alpha.html %}
     {% else %}
       <a href="{{ site.baseurl }}/discovery">Discovery</a>
+      <a href="{{ site.baseurl }}/alpha">Alpha</a>
```
Comment (JordanHatch) [inline _includes/navigation.html]: Could you remove this from the main navigation for now - whilst the guide is still in review?

File: _includes/section_header.html [hunk 2]
```diff
@@ -4,4 +4,10 @@
       <a href="{{ site.baseurl }}/discovery">Discovery</a>
     </h1>
   </header>
+  {% elsif page.section == 'alpha' %}
+  <header class="section">
+    <h1>
+      <a href="{{ site.baseurl }}/alpha">Alpha</a>
```
Comment (JordanHatch) [inline _includes/section_header.html]: There should be a 'draft' label here for the time being - we probably need to add a little bit of styling to it too.

**Comment by JordanHatch** [comment]
Thanks for your help! 
Happy for this to be merged once the two comments have been addressed. :+1:

**Comment by colugo** [comment]
Done.
Merge away

**Pull request merged**